tla-io/src/main/scala/at/forsyte/apalache/io/ReportGenerator.scala

```diff
@@ -38,6 +38,19 @@ object ReportGenerator {
     s"""<!-- Thank you for filing a report! Please ensure you have filled out all -->
       |<!-- sections, as it help us to address the problem effectively. -->
       |
+      |<!-- NOTE: Please try to ensure the bug can be produced on the latest release of -->
+      |<!-- Apalache. See https://github.com/informalsystems/apalache/releases -->
+      |
+      |## Description
+      |
+      |<!-- A clear and concise description of what the bug is. If you report an -->
+      |<!-- exception with a stack trace, no bug explanation is needed. -->
```

Comment:
Reports auto-generated by Apalache will always come with a stack trace, I don't think you need this field here.

Reply:
Makes sense. f04b798d3